Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1646,7 +1647,8 @@ void CGContextFillRects(CGContextRef context, const CGRect* rects, size_t count)
 */
 void CGContextDrawPath(CGContextRef context, CGPathDrawingMode mode) {
     NOISY_RETURN_IF_NULL(context);
-    UNIMPLEMENTED();
+
+    __CGContextDrawGeometry(context, _getAndClosePathGeometry(context->Path()), mode);
```

Comment:
>_getAndClosePathGeometry [](start = 37, length = 24)

```
if(!HasPath)
// we can abort
```

Otherwise, `Path()` will create an empty path just to draw it